WordPress/Docs/WP/DeprecatedFunctionsStandard.xml

```diff
@@ -0,0 +1,19 @@
+<documentation title="Deprecated Functions">
+    <standard>
+    <![CDATA[
+    Please refrain from using deprecated WordPress functions. 
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: not using deprecated functions.">
+        <![CDATA[
+function get_sites()
```

Comment:
This should be a _call_ to `get_sites()`, not trying to define the function. Change it to something like:

```php
$sites = get_sites();
```